### ISSUE TITLE
AH remote access UI is a bit odd

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode#322982]
Some general feedback:

1. Clicking it and immediately launching me to sign in is a bit odd
2. We cannot use just color to distinguish states for accessibility reasons
3. Yellow seems a bit odd for 'on' as I associate yellow with warning states

**Comment by connor4312** [comment]
Hm, as an alternative icon we could use the disconnect/connect "plug", although to me that more means that I want to connect _this_ window to something, not 'broadcast' my window for other people to connect to. @hawkticehurst thoughts?

**Comment by hawkticehurst** [comment]
> although to me that more means that I want to connect this window to something, not 'broadcast' my window for other people to connect to

Yeah I agree that the plug icon is probably not be the best choice here. I wonder if we could instead create a new variant of the tower icon without the "radio wave" / "wifi wave" bands, so when clicked you can toggle between no bands and with bands icons? 

Assuming we like this idea, @mrleemurray could you kick off a request with David H to create this and get it merged into Codicons repo?

> Yellow seems a bit odd for 'on' as I associate yellow with warning states

Yeah this is a fair call out. Perhaps we could go with a classic blue or even just pick the accent color of whatever theme you are using?

**Comment by connor4312** [comment]
I agree with swithcing to blue. I used the dandelion color originally since that's what we used in the secondary chat bar to indicate Autopilot mode is a super cool feature 😎 and I wanted to be cool too

**Comment by hawkticehurst** [comment]
> I agree with swithcing to blue. I used the dandelion color originally since that's what we used in the secondary chat bar to indicate Autopilot mode is a super cool feature 😎 and I wanted to be cool too

Lol oh yeah, if I was to take off my "responsible designer" hat, I'd say totally go with the yellow. I personally love that specific shade and how it looks in the workbench 😍 But alas yellow does usually mean warning so I gotta recommend switching to blue
